crml/sylo/src/groups/default_weights.rs

```diff
@@ -21,29 +21,37 @@
 use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
 
 impl crate::groups::WeightInfo for () {
-	fn create_group(i: usize) -> Weight {
-		(45_728_000 as Weight)
-			.saturating_add((59_625_000 as Weight).saturating_mul(i as Weight))
+	fn create_group(i: usize, m: usize) -> Weight {
+		(0 as Weight)
```

Comment:
why it changed from 45_728_000 as weight to 0 as weight?

Reply:
Benchmark changed it. The latest results are affected by having the maximum number of meta for the group.